client/me/purchases/manage-purchase/index.jsx

```diff
@@ -649,12 +659,13 @@ export default connect( ( state, props ) => {
 		purchase,
 		purchaseAttachedTo,
 		siteId,
+		isProductOwner,
 		site,
 		renewableSitePurchases,
 		plan: isPurchasePlan && applyTestFiltersToPlansList( purchase.productSlug, abtest ),
 		isPurchaseTheme,
 		theme: isPurchaseTheme && getCanonicalTheme( state, siteId, purchase.meta ),
 		isAtomicSite: isSiteAtomic( state, siteId ),
-		userId: getCurrentUserId( state ),
+		userId,
```

Comment:
I don't think you need to pass `userId` as a prop to the component as it's not used anywhere (that I can see) outside of this function.

Reply:
Good catch! thank you. I'll take it out.